packages/forms-web-app/src/lib/appeals-api-wrapper.js

```diff
@@ -276,6 +277,38 @@ exports.getUserAppealsById = async (id) => {
 	return handler(`/api/v2/users/${id}/appeals`, 'GET');
 };
 
+/**
+ * @typedef {import('appeals-service-api').Api.AppealCaseWithAppellant} AppealCaseWithAppellant
+ */
+
+/**
+ * @param {string} email
+ * @param {string} appealSqlId
+ * @param {string} [role]
+ * @returns {Promise<AppealCaseWithAppellant>}
+ */
+exports.linkUserToV2Appeal = async (email, appealSqlId, role) => {
```

Comment:
Thoughts on a class-based approach for v2 API calls?

e.g. https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3334/files#diff-9727715fc7c722a6ac0258c1eddf1a7e9445e6bad6047711d142f81066bc8b6c

Reply:
Definitely preferable, want me to wait for that?

Reply:
I'll bring a copy in